drivers/soft_uart/include/soft_uart_params.h

```diff
@@ -19,6 +19,7 @@
 #ifndef SOFT_UART_PARAMS_H
 #define SOFT_UART_PARAMS_H
 
+#include "board.h"
```

Comment:
This is not exactly related to this PR, but without it `SOFT_UART_PARAM_xx` set in `board.h` would not get applied.

I first thought of using the software UART to invert the line polarity, but then I found I can use a real UART instead.